### PR TITLE
Updated documentation for nb cli functionality in similar method as nb --help.

### PR DESCRIPTION
**Pull request by mbebic**
This pr links to issue #633 .

### CODE REVIEW

**Comment by idomic** [comment]
@mbebic can you please rebase on the master to make sure the CI is passing?
Also, make sure you include all of the nb functionality, I've seen some weren't included, like pair, sync, format etc. You can view the commands via the `ploomber nb --help`

File: doc/api/cli.rst [hunk 1]
```diff
@@ -300,25 +300,38 @@ your code. To install from lock files:
 
 ``nb`` is short for *notebook*. This command manages notebooks and scripts in your pipeline.
 
+Inject cell to scripts and notebooks in your pipeline:
+
+.. code-block:: console
+
+    ploomber nb --inject
```
Comment (idomic) [inline doc/api/cli.rst:307]: Make sure you include the rest of the functionalities of this command

**Review by idomic — CHANGES_REQUESTED** [review]
Please fix and recommit

**Comment by edublancas** [comment]
thanks for your contribution!

**Pull request merged**